WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingAddresses/WooShippingEditAddressView.swift

```diff
@@ -77,6 +84,30 @@ struct WooShippingEditAddressView: View {
                 }
             }
         }
+        .safeAreaInset(edge: .bottom) {
+            VStack(spacing: .zero) {
+                Divider().ignoresSafeArea(edges: [.horizontal])
+                VStack(spacing: Constants.verticalSpacing) {
+                    HStack {
+                        Image(systemName: viewModel.status == .verified ? "checkmark.circle" : "exclamationmark.circle")
+                        Text(Localization.Status.label(for: viewModel.status))
+                    }
+                    .font(.subheadline)
+                    .foregroundStyle(viewModel.status == .verified ? Constants.green : Constants.red)
+                    Button(Localization.Button.label(for: viewModel.status)) {
+                        if viewModel.status == .verified {
+                            dismiss()
+                        } else {
```

Comment:
Do we also have to handle the case when information is missing?

Reply:
We don't have to (the button is disabled in that case) but we could add a `break` just to explicitly cover the case. I've made a note to come back to that when I add the handling for missing information to the view model.